apps/projects/test/projects.test.js

```diff
@@ -136,7 +168,7 @@ contract('Projects App', accounts => {
     )
 
     // Deploy test Bounties contract
-    bounties = await StandardBounties.new(web3.toBigNumber(owner1))
+    bounties = { address: '0x72D1Ae1D6C8f3dd444b3D95bAd554Be483082e40'.toLowerCase() }
```

Comment:
the test for changing bounty settings fails:

```
1) Contract: Projects App
       post-initialization
         settings management
           can change Bounty Settings:

      StandardBounties Contract address incorrect
      + expected - actual

      -0x72d1ae1d6c8f3dd444b3d95bad554be483082e40
      +0x72D1Ae1D6C8f3dd444b3D95bAd554Be483082e40
```

Reply:
Ok I think I understand, to prevent different hashes for the same address with different casing, yeah, thinking about it I agree is a good decision, it seems less cumbersome than dealing with checksummed addresses 👍